test/12-fancy-cmd/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -ex
+
+(cd ../../src/cmd/makebb && GO111MODULE=on go build .)
+MAKEBB=../../src/cmd/makebb/makebb
+
+# This command intentionally starts with digits and contains characters not
+# valid in identifiers (`-`).
+for GO111MODULE in on auto;
+do
+  if GO111MODULE=$GO111MODULE $MAKEBB .; then
+    echo "makebb is happy"
```

Comment:
can you also run ./bb 12-fancy-cmd to make sure the right command name is mapped despite the rename? (check for the "12-fancy-cmd" output)

Reply:
done :)